### PR TITLE
Updating sloting function for promoting wikis

### PR DESCRIPTION
**Pull request by codiejay**
- [x] allow full urls on revalidate widget
- [x] make promote work correctly
- [x] make archive / unarchive perform a revalidate action after graph is called 

Currently the revalidation after promoting wikis doesn't happen ON this PR link but works fine on local and when we manually revalidate the PR link after promoting a wiki.

Will demo this over a live call with @Softdev1 . 

closes https://github.com/EveripediaNetwork/issues/issues/815

### CODE REVIEW

**Comment by kesar** [comment]
lets get a review and merge if works as expected for this plz cc: @Softdev1 @Royal-lobster

**Review by Softdev1 — CHANGES_REQUESTED** [review]
The promote wiki is not still working as expected @codiejay

**Comment by codiejay** [comment]
> The promote wiki is not still working as expected @codiejay

Working on it. This is just a bit weird cause locally everything works just fine.

**Comment by Softdev1** [comment]
This would better help to guide on what is really happening with it. When a wiki is been promoted, on the admin panel it shows that it is promoted with the right tag but on the UI side of things you get incorrect info.  it seems the revalidate wiki is not working well for the promoted wiki
this is not working at all

**Comment by kesar** [comment]
whats the status of this? still in development ?

**Comment by Softdev1** [comment]
@kesar  he is not done fixing this issue that is why it is still opened sir @kesar  , I still reached out to him about this

File: src/components/Admin/WikiCreatedInsight/PromoteCreatedWikisModal.tsx [hunk 1]
```diff
@@ -250,12 +251,16 @@ export const PromoteCreatedWikisModal = ({
...
+        const id = getWikiIdUsingLevel(+value)
+        if (id) {
+          handlePromoteWiki({ id, level: 0 })
+          promoteWikiRefetch()
+        }
         hideFunc()
+        promoteWikiRefetch()
         Close()
         let toastTitle = 'Wiki Successfully Promoted to Trending wikis'
         let toastMessage =
```
Comment (Royal-lobster) [inline src/components/Admin/WikiCreatedInsight/PromoteCreatedWikisModal.tsx:266]: @codiejay we can move the toast logic to the `handlePromoteWiki` function to make the code dry

**Review by Royal-lobster — CHANGES_REQUESTED** [review]

**Review by Royal-lobster — APPROVED** [review]

**Review by Royal-lobster — APPROVED** [review]

**Comment by kesar** [comment]
letss gooo

**Pull request merged**